packages/asc-ui/src/components/TextArea/TextArea.tsx

```diff
@@ -0,0 +1,58 @@
+import React, { useImperativeHandle } from 'react'
+import useActionOnEscape from '../../utils/hooks/useActionOnEscape'
+import useFocusOnRender from '../../utils/hooks/useFocusOnRender'
+import TextAreaStyle from './TextAreaStyle'
+
+export interface TextAreaMethods
```

Comment:
`setTextAreaRef` is never used, let's remove `TextAreaMethods` entirely.

Reply:
done